checks/test/ChangelogTest.test.scala

```diff
@@ -0,0 +1,258 @@
+import Dependencies.*
+import upickle.default.*
+import ChangelogTest.*
+import scala.concurrent.duration.Duration
+import ox.*
+
+class ChangelogTest extends munit.FunSuite:
```

Comment:
Since this file is in a `test` directory, I don't think it should have the `.test` suffix

Reply:
Right! I had some problems with that, but now structure is different and it works ok